main.tf

```diff
@@ -54,14 +54,16 @@ resource "aws_sqs_queue" "queue" {
   count = "${length(var.kms_alias) == 0 ? 1 : 0}"
   name  = "${var.name}"
 
-  visibility_timeout_seconds  = "${var.visibility_timeout_seconds}"
-  message_retention_seconds   = "${var.message_retention_seconds}"
-  max_message_size            = "${var.max_message_size}"
-  policy                      = "${var.policy}"
-  delay_seconds               = "${var.delay_seconds}"
-  receive_wait_time_seconds   = "${var.receive_wait_time_seconds}"
-  fifo_queue                  = "${var.fifo_queue}"
-  content_based_deduplication = "${var.content_based_deduplication}"
+  visibility_timeout_seconds        = "${var.visibility_timeout_seconds}"
+  message_retention_seconds         = "${var.message_retention_seconds}"
+  max_message_size                  = "${var.max_message_size}"
+  policy                            = "${var.policy}"
+  delay_seconds                     = "${var.delay_seconds}"
+  receive_wait_time_seconds         = "${var.receive_wait_time_seconds}"
+  fifo_queue                        = "${var.fifo_queue}"
+  content_based_deduplication       = "${var.content_based_deduplication}"
+  kms_master_key_id                 = "${aws_kms_key.sqs_kms_key.key_id}"
+  kms_data_key_reuse_period_seconds = 300
```

Comment:
This doesn't really make sense. You've already got a sqs resource with kms encryption below so why duplicate it here?